go.mod

```diff
@@ -1,75 +1,35 @@
-	github.com/hashicorp/go-immutable-radix v1.0.0
-	github.com/hashicorp/golang-lru v0.5.0
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/gobuffalo/packr/v2 v2.7.1
+	github.com/hashicorp/go-immutable-radix v1.1.0
+	github.com/hashicorp/golang-lru v0.5.3
 	github.com/jacobsa/daemonize v0.0.0-20160101105449-e460293e890f
-	github.com/jacobsa/fuse v0.0.0-20180417054321-cd3959611bcb
-	github.com/json-iterator/go v1.1.6
-	github.com/karrick/godirwalk v1.10.12
-	github.com/kylelemons/godebug v1.1.0 // indirect
-	github.com/magiconair/properties v1.8.1 // indirect
-	github.com/markbates/oncer v0.0.0-20181203154359-bf2de49a0be2 // indirect
-	github.com/markbates/safe v1.0.1 // indirect
-	github.com/matoous/godox v0.0.0-20190911065817-5d6d842e92eb // indirect
-	github.com/mattn/go-colorable v0.1.4 // indirect
-	github.com/mattn/go-isatty v0.0.9 // indirect
+	github.com/jacobsa/fuse v0.0.0-20190923155423-081e9f4bc7d4
+	github.com/matryer/moq v0.0.0-20190312154309-6cfb0558e1bd // indirect
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
 	github.com/nightlyone/lockfile v0.0.0-20180618180623-0ad87eef1443
-	github.com/opentracing/opentracing-go v1.0.2
-	github.com/pelletier/go-toml v1.4.0 // indirect
-	github.com/securego/gosec v0.0.0-20191002120514-e680875ea14d // indirect
+	github.com/opentracing/opentracing-go v1.1.0
+	github.com/rogpeppe/go-internal v1.5.0 // indirect
 	github.com/segmentio/ksuid v1.0.2
-	github.com/spf13/afero v1.2.2
+	github.com/spf13/afero v1.1.2
 	github.com/spf13/cobra v0.0.5
-	github.com/spf13/jwalterweatherman v1.1.0 // indirect
-	github.com/spf13/viper v1.4.0
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/spf13/viper v1.3.2
 	github.com/stretchr/testify v1.4.0
-	github.com/timakin/bodyclose v0.0.0-20190930140734-f7f2e9bca95e // indirect
-	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8 // indirect
-	github.com/ultraware/whitespace v0.0.4 // indirect
-	go.uber.org/goleak v0.0.0-00010101000000-000000000000
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/goleak v0.10.0
+	go.uber.org/multierr v1.2.0 // indirect
 	go.uber.org/zap v1.10.0
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e // indirect
-	golang.org/x/sys v0.0.0-20191003212358-c178f38b412c
-	golang.org/x/tools v0.0.0-20191004211743-43d3a2ca2ae9
-	google.golang.org/api v0.2.0
-	google.golang.org/grpc v1.21.0
-	gopkg.in/yaml.v2 v2.2.4
-	gotest.tools/gotestsum v0.3.5 // indirect
-	k8s.io/apimachinery v0.0.0-20190531161113-d9689afd32c1 // indirect
-	k8s.io/kubernetes v1.14.2
-	k8s.io/utils v0.0.0-20190520173318-324c5df7d3f0 // indirect
-	mvdan.cc/unparam v0.0.0-20190917161559-b83a221c10a2 // indirect
-	sourcegraph.com/sqs/pbtypes v1.0.0 // indirect
+	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550 // indirect
+	golang.org/x/sys v0.0.0-20191010194322-b09406accb47
+	golang.org/x/tools v0.0.0-20191015211201-9c6d90b5a7d0 // indirect
+	google.golang.org/api v0.11.0
+	gopkg.in/yaml.v2 v2.2.2
```

Comment:
why rollbacking yaml to 2.2.2? IIRC, 2.2.4 brought a security fix.

Reply:
Thank you